### ISSUE TITLE
Riot services and method signatures

### ISSUE THREAD

**Issue by desmond-silveira** [achimala/leaguelib#15]
Does anyone know what Riot service and method signature is used to retrieve honor information?  If someone knows, I can code that up.

More generally, how are _any_ of the services and method signatures determined?

**Comment by tyxieblub** [comment]
The service is "clientFacadeService" and the method name is "callKudos". The argument for this is an array with one element, which is a String built as following : 

``` java
"{\"summonerId\":" + summonerId + ",\"commandName\":\"TOTALS\"}"
```

Just be sure to use the summonerId and not the accountId.

If I'm not mistaken, these are found by using man in the middle programs.

**Comment by tyxieblub** [comment]
Oh well, I quickly implemented it. If you're still needing it, you can check this commit : 4573b41a5d2854ec61692897a1a702e1d5d475d8

**Comment by desmond-silveira** [comment]
Thanks!  I'm using your commit.  Great work.

There were two very minor improvements to it that I commented on in the commit.

**Comment by desmond-silveira** [comment]
BTW, would WireShark be the best man-in-the-middle program to find the service and method signatures?

**Comment by tyxieblub** [comment]
I don't think so, Wireshark isn't really a MITM program , it's just a packet analyzer and as the connection between the client and the server is secured (since it uses RTMPS) you won't be able to see the real content in the packet. 
You should be using LoLNotes ( https://github.com/bladecoding/LoLNotes/downloads ) to see that. That said, I haven't been able to make it work recently : League of Legends is unable to connect when LoLNotes is launched, but if you manage to, you will be able to look into the messages sent between the client and the server.
